.github/workflows/test.yaml

```
@@ -0,0 +1,17 @@
+name: Pull Request
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.14.10
```

Would it make sense to read the terraform version from `provider.tf` ?
```suggestion
      - uses: actions/checkout@v2      
      - name: Determine Terraform version from provider.tf
        run: echo "TF_VERSION=$(grep "required_version = \">= " provider.tf | cut -d " " -f 2)" >> $GITHUB_ENV
      - uses: hashicorp/setup-terraform@v1
        with:
          terraform_version: ${{ env.TF_VERSION }}
```
(grep command not tested)
Provided setup-terraform can handle "0.14" version without a specific patch version part.

---

Adjusted the workflow accordingly